resources/web-environment/.gitignore

```diff
@@ -0,0 +1,2 @@
+environment.inc.ts
```

Comment:
Should this be cleaned up (removed) in the other common/core/web/**/.gitignore files?

Reply:
Eventually, but that would require each of us to do a repo-wide cleaning of this file on our development machines.  I didn't want to enforce that quite yet, as I've come rather close to accidentally committing `.gitignore`'d files when branch-hopping and would rather not 'chance' that happening.